### DISCUSSION TITLE
API endpoint for billing is returning 500 server error

### DISCUSSION THREAD

**Discussion by seantact** [meilisearch/product#774] [Cloud Feedback]
Hello!

FYI - when I go to the billing page I noticed that there is an error.. Looking in my browser console there is a 500 server error returning on this endpoint: https://api.meilisearch.com/api/v1/projects/[my project id]/usage

**Comment by curquiza** [comment]
Hello @seantact 

Can you contact the support of the Cloud directly?
It will prevent you from sharing private information in public here. We will do our best to fix the problem as quickly as possible

**Reply by curquiza** [reply]
I close this discussion since it's a bug report and not a feature request, and we need more private information to move forward on it

**Reply by curquiza** [reply]
Thank you for the report 🙏
